app/oidc/gcp_oidc.py

```diff
@@ -0,0 +1,35 @@
+from cachetools.func import ttl_cache
+from google.auth.credentials import Credentials
+from google.auth.transport.requests import Request
+from google.oauth2.id_token import fetch_id_token_credentials
+from structlog import get_logger
+
+from app.oidc.oidc import TTL, OIDCCredentialsService, refresh_oidc_credentials
+
+logger = get_logger()
+
+
+class GCPCredentialsService(OIDCCredentialsService):
```

Comment:
OIDC should be in the name for clarity.

Reply:
I did have this, and it was `GCPOIDCCredentialService` which looked like an model number or something, shall we go with that, or change the impl part to be a suffix like `OIDCCredentialServiceGCP` and `OIDCCredentialServiceLocal`?

Reply:
😄 True. Yes go with `OIDCCredentialServiceGCP` and `OIDCCredentialServiceLocal`

Reply:
Renamed 👍